keyboards/illuminati/is0/info.json

```
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "iS0", 
+    "url": "", 
+    "maintainer": "ai03", 
+    "width": 1.5, 
+    "height": 1, 
```

This is a bug in our KLE to info.json converter. Keys taller than 1u don't increase the height when they should.

```suggestion
    "height": 2, 
```